comfyui_pano_suite/comfy_image_resolver.py

```
@@ -32,14 +33,39 @@ def _resolve_comfy_image_path(asset: dict) -> Path | None:
     return path if path.exists() and path.is_file() else None
 
 
-def _load_comfy_rgba(asset: dict) -> np.ndarray | None:
+def _resolve_asset_cache_key(asset: dict | None) -> tuple[str, int, int] | None:
     path = _resolve_comfy_image_path(asset)
     if path is None:
         return None
     try:
-        return np.asarray(Image.open(path).convert("RGBA"), dtype=np.float32) / 255.0
+        stat = path.stat()
     except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)